packages/patternfly-4/react-core/src/expiremental/components/Badge/Badge.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { Badge } from './Badge';
+import React from 'react';
+import { shallow } from 'enzyme';
+
+Object.values([true, false]).forEach(isRead => {
+  test(`${isRead} Badge`, () => {
+    const view = shallow(<Badge isRead={isRead}>{isRead ? 'read' : 'unread'} Badge</Badge>);
+    expect(view).toMatchSnapshot();
```

Comment:
I'd really like to see test that check for expected values rather than relying on snapshots. That is, adding a check for the existance of the `pf-m-unread` when isRead is false and for `pf-m-read` when isRead is true.

Reply:
These components will likely be removed, they are just to test the site for now. I agree the tests should cover the experimental features.